File: test/fixtures/dart/repeated_query_key.dart
import 'package:http/http.dart' as http;

void main() async {
  final params = {
    'key': ['one', 'two'],
  };

  final url = Uri.parse('http://localhost:28139')
      .replace(queryParameters: params);

  final res = await http.get(url);
  final status = res.statusCode;
  if (status != 200) throw Exception('http.get error: statusCode= $status');

  print(res.body);
}

File: test/fixtures/swift/repeated_query_key.swift
import Foundation

let url = URL(string: "http://localhost:28139?key=one&key=two")!

var request = URLRequest(url: url)

let task = URLSession.shared.dataTask(with: request) { (data, response, error) in
    if let error = error {
        print(error)
    } else if let data = data {
        let str = String(data: data, encoding: .utf8)
        print(str ?? "")
    }
}

task.resume()
